frontend/src/modules/activity/config/filters/activityType/config.ts

```diff
@@ -13,13 +17,13 @@ const activityType: CustomFilterConfig = {
   component: ActivityTypeFilter,
   options: {
   },
-  queryUrlParser: queryUrlParserByType[FilterConfigType.SELECT],
-  itemLabelRenderer(value: SelectFilterValue, options: SelectFilterOptions, data: any): string {
-    return itemLabelRendererByType[FilterConfigType.SELECT]('Activity type', value, data, null, {
+  queryUrlParser: queryUrlParserByType[FilterConfigType.MULTISELECT],
+  itemLabelRenderer(value: MultiSelectFilterValue, options: MultiSelectFilterOptions, data: any): string {
+    return itemLabelRendererByType[FilterConfigType.MULTISELECT]('Activity type', value, data, null, {
       addGroupLabel: true,
     });
   },
-  apiFilterRenderer({ value, include }:SelectFilterValue): any[] {
+  apiFilterRenderer({ value, include }:MultiSelectFilterValue): any[] {
     const [platform, activityType] = value.split(':');
```

Comment:
I'm getting a console.error when I apply this filter, so it's not even being applied.

main.ts:66 TypeError: value.split is not a function or its return value is not iterable
    at Object.apiFilterRenderer (config.ts:27:44)
    at filter-api.service.ts:52:30
    at Array.forEach (<anonymous>)
    at buildApiFilter (filter-api.service.ts:47:34)
    at fetch (Filter.vue:116:3)
    at watch.deep (Filter.vue:121:3)
    at callWithErrorHandling (runtime-core.esm-bundler.js:158:18)
    at callWithAsyncErrorHandling (runtime-core.esm-bundler.js:166:17)
    at job (runtime-core.esm-bundler.js:1756:9)
    at callWithErrorHandling (runtime-core.esm-bundler.js:158:32)
Can you take a look?

Also, can you refactor the logic so that we filter with an or operator?
There will never be one activity with multiple activity types, so we should filter out activities that can contain each one